### PR TITLE
Fixes #33715 - implement Log in React component

### PR DESCRIPTION
**Pull request by ezr-ondrej**
Implements the Log console in react component.

### CODE REVIEW

File: app/views/hosts/console/log.html.erb [hunk 1]
```diff
@@ -3,12 +3,5 @@
...
 </div>
-<% if @host && (@host.installed_at.nil? or (@console['timestamp'] - @host.installed_at < 5.minutes)) %>
-  <%= alert :class => 'alert-info', :text => _('Console output may be out of date') %>
-<% end %>
-<pre class="pre-scrollable">
-  <code>
-    <%= @console['output'] %>
-  </code>
-</pre>
+<%= react_component('LogConsole', @console.merge(outdated: @host && (@host.installed_at.nil? or (@console['timestamp'] - @host.installed_at < 5.minutes)))) %>
```
Comment (ekohl) [inline app/views/hosts/console/log.html.erb:6]: I don't see `@console['output']` being passed anywhere. How does it get the output?

Comment (ezr-ondrej) [reply app/views/hosts/console/log.html.erb:6]: `@console` is a hash, so it gets translated to props directly (whatever keys it has will become props) and `output` happens to be one of the keys.

File: webpack/assets/javascripts/react_app/components/Console/Log.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Alert } from 'patternfly-react';
+
+import { translate as __ } from '../../common/I18n';
+
+const Log = ({ output, timestamp, outdated }) => (
```
Comment (ekohl) [inline webpack/assets/javascripts/react_app/components/Console/Log.js:7]: timestamp is unused

File: app/views/hosts/console/log.html.erb (hunk 1 above)
Comment (ekohl) [reply app/views/hosts/console/log.html.erb:6]: Ah, that's well hidden. I'd not couple the APIs so directly and be explicit in what you pass.

**Pull request closed without merging**